### DISCUSSION TITLE
A way to make the later command execute in the same tab it was entered

### DISCUSSION THREAD

**Discussion by kiil** [qutebrowser/qutebrowser#6328] [Ideas] [answered]
Hi - great and interesting browser :)

The `later` command is interesting. But  is there  any way to make sure that its command is actually executed in the tab it was given in?

Now, if I change to another tab before the duration has expired, the command will be executed in whatever tab I am currently in - instead of in the tab I gave the command in.

Regs, Lennart

**Comment by kiil** [comment]
One way around this is to use the later command in a window with only one tab.

**Comment by The-Compiler** [comment, marked as answer]
Sorry for the radio silence, this got lost in my inbox for a bit. I opened #6370 with a couple of related ideas. Out of curiosity, what's your use case?

**Reply by kiil** [reply]
Thanks for replying :)

My use case is something like this:

```
/Applications/qutebrowser.app/Contents/MacOS/qutebrowser https://www.somedomainsomewhere.com/admin/config/services/mailchimp/lists/update_mergevars/user/user/field_nyhedsbrev ":later 1500 click-element id edit-submit;;later 60000 close" --target window
```

And it works :)
